appium-tests/common/common.spec.js

```diff
@@ -242,7 +248,6 @@ describe('Contacts Android', function () {
 
     describe('Picking contacts', function () {
         afterEach(function (done) {
-            checkSession(done);
```

Comment:
I still left the removal of this check, @alsorokin, as I don't think it makes sense to put in the `afterEach`. With the spec itself invoking `checkSession`, if the appium session failed to start, it would get caught in the spec. In fact, we could probably get away with checking the session only immediately after the initial `should connect to an appium endpoint properly` spec. Any further checks beyond that would not behave any differently.